### PR TITLE
[DO NOT MERGE] design preview

### PR DESCRIPTION
**Pull request by thomasheyenbrock**
The purpose of this PR is just to generate a stable Netlify preview deployment that shows the latest efforts on the new design implementation.

### CODE REVIEW

**Comment by acao** [comment]
@thomasheyenbrock the canary release tags change on every push fyi!

File: packages/graphiql-toolkit/src/storage/base.ts [hunk 1]
```diff
@@ -77,6 +78,12 @@ export class StorageAPI {
 
     return { isQuotaError: quotaError, error };
   }
+
+  clear() {
+    if (this.storage) {
+      this.storage.clear();
+    }
```
Comment (B2o5T) [inline packages/graphiql-toolkit/src/storage/base.ts:85]:
```suggestion
    this.storage?.clear();
```

File: packages/graphiql/src/components/DocExplorer.tsx [hunk 2]
```diff
@@ -106,41 +75,33 @@ export function DocExplorer(props: DocExplorerProps) {
...
-              props.onClose?.();
-            }}
-            aria-label="Close Documentation Explorer">
-            {'\u2715'}
-          </button>
+      <div className="graphiql-doc-explorer-header">
+        <div className="graphiql-doc-explorer-header-content">
+          {prevName && (
+            <a
+              href="#"
```
Comment (B2o5T) [inline packages/graphiql/src/components/DocExplorer.tsx:84]: I guess for accessibility will be better to use `button` instead `a`

File: packages/graphiql/src/components/DocExplorer/FieldDoc.tsx [hunk 3]
```diff
@@ -5,127 +5,117 @@
...
-      {deprecatedArgsDef}
-    </div>
+    <>
+      {args.length > 0 ? (
+        <ExplorerSection title="Arguments">
+          {args.map(arg => (
+            <Argument key={arg.name} arg={arg} />
+          ))}
+        </ExplorerSection>
+      ) : null}
```
Comment (B2o5T) [inline packages/graphiql/src/components/DocExplorer/FieldDoc.tsx:81]:
```suggestion
      {args.length > 0 && (
        <ExplorerSection title="Arguments">
          {args.map(arg => (
            <Argument key={arg.name} arg={arg} />
          ))}
        </ExplorerSection>
      )}
```

File: packages/graphiql/src/components/DocExplorer/FieldDoc.tsx [hunk 4]
```diff
@@ -5,127 +5,117 @@
...
+          </ExplorerSection>
+        ) : (
+          <Button
+            onClick={() => {
+              setShowDeprecated(true);
+            }}>
+            Show deprecated arguments
+          </Button>
+        )
+      ) : null}
```
Comment (B2o5T) [inline packages/graphiql/src/components/DocExplorer/FieldDoc.tsx:97]:
```suggestion
      {deprecatedArgs.length > 0 && (
        showDeprecated || args.length === 0 ? (
          <ExplorerSection title="Deprecated Arguments">
            {deprecatedArgs.map(arg => (
              <Argument key={arg.name} arg={arg} />
            ))}
          </ExplorerSection>
        ) : (
          <Button
            onClick={() => {
              setShowDeprecated(true);
            }}>
            Show deprecated arguments
          </Button>
        )
      )}
```

File: packages/graphiql/src/components/GraphiQL.tsx [hunk 5]
```diff
@@ -21,13 +21,32 @@ import {
 } from 'graphql';
 
 import {
+  Button,
+  ButtonGroup,
+  ChevronDownIcon,
+  ChevronUpIcon,
```
Comment (B2o5T) [inline packages/graphiql/src/components/GraphiQL.tsx:27]: A tip, normally you don't need to have 2 same icons, you can use CSS for rotate your icon :)

**Comment by thomasheyenbrock** [comment]
@B2o5T thanks for reviewing this 🙏  I'll probably split this huge PR up into smaller increments though to make the individual changes even more reviewable 😉

**Comment by thomasheyenbrock** [comment]
All changes have been split up and reviewed incrementally. We collected all of them on the `graphiql-v2` branch, this PR is basically succeeded by #2523.

**Comment by acao** [comment]
Marvelous work thank you!

**Pull request closed without merging**